### PR TITLE
Moving to az do live subscription

### PR DESCRIPTION
**Pull request by elduddz**
- using makefile for process management
- experimenting with some secrets being encrypted in repo
- using latest "stable" packer image
- using F8s whilst we work out why F8s_V2 arent on the subscription

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -10,20 +10,16 @@ pr: none
 
 pool:
   vmImage: 'ubuntu-latest'
-
+container: ant59/terraform-azure-make:latest
 steps:
+- bash: ls -lah
```
Comment (MarcDenman) [inline azure-pipelines.yml]: Needed?

File: modules/azdo_ws2019agent/main.tf [hunk 2]
```diff
@@ -1,8 +1,8 @@
 data "azurerm_shared_image_version" "existing" {
-  name                = "0.153.3"
-  gallery_name        = "UKHOSharedImageGallery"
+  name                = "0.154.3"
+  gallery_name        = "UkhoSharedImageGallery"
```
Comment (MarcDenman) [inline modules/azdo_ws2019agent/main.tf]: Can UKHO be capitalised?

File: tfplan.sh [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+
+chmod 755 ./secret
 
-terraform init
+
+make init
 
 echo "Execute Terraform plan"
-terraform plan -out=tfplan -input=false
+#terraform plan -out=tfplan -input=false
```
Comment (MarcDenman) [inline tfplan.sh]: remove if not needed

File: tfplan.sh [hunk 4]
```diff
@@ -1,15 +1,25 @@
...
+make plan
 
 echo "Execute Terraform apply"
-terraform apply "tfplan"
+#terraform apply "tfplan"
```
Comment (MarcDenman) [inline tfplan.sh]:
```suggestion

```

**Review by MarcDenman — COMMENTED** [review]
Can `./iptest-pipelines.yml` be deleted? Is the documentation up to date? Are there any unit tests?

File: modules/azdo_ws2019agent/main.tf (hunk 2 above)
Comment (elduddz) [reply modules/azdo_ws2019agent/main.tf]: I dont think it matters, the resource I created was UkhoSharedImageGallery

**Comment by elduddz** [comment]
> Can `./iptest-pipelines.yml` be deleted? Is the documentation up to date? Are there any unit tests?

not sure what can I unit test?

**Review by MarcDenman — APPROVED** [review]

**Pull request merged**